x/auth/migrations/v4/migrate.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/auth/types"
 )
 
-var ParamsKey = []byte{0x01}
+var ParamsKey = []byte{0x00}
```

Comment:
well this is concerning

Reply:
we need way better testing on migrations. This is too close to a release